lib/phobos_db_checkpoint/handler.rb

```diff
@@ -31,7 +45,15 @@ def around_consume(payload, metadata)
           end
 
           event_action = instrument('db_checkpoint.event_action', event_metadata) do
-            yield
+            begin
```

Comment:
I know @ticolucci has comments about this being a separate handler.

But I think error handling is mandatory, less classes and less confusion. You could always configure retry_consume? to always return true for example, to escape this functionality and restore the behaviour before having error handling.

WDYT?